### ISSUE TITLE
Doesn't open location in Apple Maps

### ISSUE THREAD

**Issue by jzxchiang1** [flexible-agency/react-native-map-link#222]
When I click on the link and select Apple Maps from the popup on iOS, it switches to Apple Maps and displays my current location but NOT the lat/long location I passed to `showLocation` like such:

```
      <Button
        title="Open Maps"
        onPress={() =>
          showLocation({
            latitude: 37.3114919,
            longitude: -122.0657608,
            googlePlaceId: 'ChIJRQOCvWa0j4ARlxoULrHKCmg',
            alwaysIncludeGoogle: true,
            appsWhiteList: ['google-maps', 'apple-maps'],
          })
        }
      />
```

I am testing on an iPhone 13, iOS 15.2 simulator, 2.9.3 version of this library. It works flawlessly with Google Maps. Anyone know what is the issue?

**Comment by fcandi** [comment]
I had the same problem.

I do not have the time to dig deeper, but I found a simple preliminary fix: simply also set the title to "lat,lng". Its not pretty, but it works for now :)

**Comment by Kjagd** [comment]
We are also seeing Apple Maps just searching for the Title and ignoring the lat/lng. Not sure if this is somehow new behaviour from Apples side?

**Comment by trooperandz** [comment]
I am seeing the same thing. Apple Maps will open the searched map location correctly when `title` is passed, but if you only pass the `latitude` and `longitude`, it will ignore that and use your current location instead.

**Comment by Kjagd** [comment]
Seems like in this PR https://github.com/flexible-agency/react-native-map-link/pull/207 the query was changed to use the `sll` param. But according to the docs, this is a search location, ie. the query http://maps.apple.com/?q=Mexican+Restaurant&sll=50.894967,4.341626&z=10&t=s will find Mexican restaurants around the coordinate. 

So I don't think the current implementation is correct, at least not for navigating to a specific coordinate with a custom label. @mgithubmessier

**Comment by tschoffelen** [comment]
Fixed in #224

**Comment by mgithubmessier** [comment]
I feel that we should be allowing the user to provide the `sll` query parameter -- a localized search even for normal street address text is very useful because you can't provide a title to it if you only provide lat/lng at this junction. @tschoffelen how would you feel about it if I added an option called `appleLocalizedLatLon` as well, which replaces instead of ignoring the existing latlong options? That way we can have our cake and eat it too for the folks who want to provide a specific latlon
